lldb/include/lldb/Target/Process.h

```diff
@@ -1380,6 +1380,8 @@ class Process : public std::enable_shared_from_this<Process>,
 
   virtual bool GetProcessInfo(ProcessInstanceInfo &info);
 
+  virtual lldb_private::UUID FindBuildId(const llvm::StringRef path);
+
```

Comment:
There is already a function in Process.h that can do this:
```
  virtual bool GetModuleSpec(const FileSpec &module_file_spec, const ArchSpec &arch, ModuleSpec &module_spec);
```
Remove this and use that call.